### DISCUSSION TITLE
adding folder searching priority to dropbox plugin

### DISCUSSION THREAD

**Discussion by Astrolobos** [Flow-Launcher/Flow.Launcher#2596] [Ideas]
moving this from Q&A to Ideas

I'm an artist and I use dropbox to keep all my references accessible at all time so the dropbox plugin has really helped speed up my workflow, the problem I'm having is that sometimes the artist's name is on a many of the files within a folder so when trying to open that folder flow launcher is filled with a bunch of image files instead.
What I'm asking is would it be possible to add a priority list for files vs. folders or for file types to the dropbox plugin?

**Comment by jjw24** [comment]
Hi there, it sounds like you are after the sorting functionality, this would be best asked over at the plugin's repo and as an issue. You should be able to find the repo link inside the plugin settings underneath where the icons sit. We don't maintain the Dropbox plugin unfortunately.
